src/requestHandlers/errorHandler.js

```diff
@@ -0,0 +1,30 @@
+const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function errorHandler(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message: 'Text to convert is required. Correct request is: '
+        + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!availableCases.includes(targetCase) && targetCase) {
+    errors.push({
+      message: 'This case is not supported. Available cases: '
+        + 'SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
```

Comment:
```suggestion
        + availableCases.join(", "),
```

Reply:
Or you can also use some object to store by some keys Error text and just show here it.
```
const VALIDATION_ERRORS = {
  NO_TEXT: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
  NO_TARGET_CASE: '"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase= 
 <CASE_NAME>".',
  CASE_NOT_SUPPORTED: 'This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
};
```